README.md

```diff
@@ -5,11 +5,6 @@ A Flutter plugin to manage the device's screen on Android and iOS.
 ## Usage
 To use this plugin, add `screen` as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/).
 
-Make sure you add the following permissions to your Android Manifest
-```
-<uses-permission android:name="android.permission.WAKE_LOCK" />
-```
-
```

Comment:
Screen plugin can be used without using the wakelock feature. For example to use just the getBrightness or setBrightness method.